viewing_party/party.py

```diff
@@ -1,23 +1,201 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        print("hello")
+        dict = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return dict
+
+    return None

```

Comment:
This function looks nice! Your logic here does accomplish returning None when title/genre/rating are not Truthy. However, you could make your intent more explicit so that another person reading this would quickly pick up that "when title/genre/rating are **not** Truthy this function returns None". 

You could do that by putting `return None` into an else block. 

Also, don't forget to return stray debugging print statements (line 5).